boards/ble400/dist/openocd.cfg

```diff
@@ -0,0 +1,5 @@
+source [find interface/jlink.cfg]
+transport select swd
```

Comment:
Neither do I. Both `swd` and `hla_swd` seem to work. I can change it to `hla_swd`